datatypes/src/error.rs

```diff
@@ -212,6 +211,12 @@ pub enum Error {
     NoMatchingVectorDataTypeForOgrGeometryType,
 
     NoMatchingFeatureDataTypeForOgrFieldType,
+
+    InvalidProjDefinition,
```

Comment:
why not add the projection?

Reply:
3ae7a825079e9e6b720dd7a8badb99d6b2ac6e0f